thunder/__init__.py

```diff
@@ -368,6 +368,7 @@ def _alias_tensor_of_args_kwargs_dict(*args, **kwargs) -> dict[int, list[int]]:
                 tensor_group_index_to_tensor_indices[tgi].append(idx)
         return tensor_group_index_to_tensor_indices
 
+    @thunder.core.profile.profile("_alias_tensor_of_args_kwargs")
```

Comment:
This function doesn't do any profiling. Why do you call it "profile"? Do you intend to add routing to [torch.profiler.profile](https://pytorch.org/docs/stable/profiler.html#torch.profiler.profile)?

Reply:
🤷 I think of adding nvtx markers as "profiling".

But I don't have any stake in the naming here--what would be a good name, do you think?

Reply:
To your second question:

> Do you intend to add routing to [torch.profiler.profile](https://pytorch.org/docs/stable/profiler.html#torch.profiler.profile)?

I didn't intend to, but you and TomV think that'd be useful I'd be happy to do so.

Reply:
Let's name it `annotate_for_profiling`